test/template_connector/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.0'
+
+services:
+  # TODO: add a service for the platform you want to connect with secretless
```

Comment:
The wording here is odd - we don't want to connect a service to secretless but instead connect secretless to a service